include/librealsense2/dds/dds-participant.h

```diff
@@ -38,8 +39,27 @@ class dds_participant
     eprosima::fastdds::dds::DomainParticipant * get() const { return _participant; }
     eprosima::fastdds::dds::DomainParticipant * operator->() const { return get(); }
 
+    // Every participant in DDS has a global identifier, comprising a prefix- and entity-ID.
+    // The prefix (vendor, host, process, participant) usually does not vary within the same participant.
```

Comment:
The prefix does not vary in FastDDS implementation, not guarantied for other vendors or even SafeDDS.

Reply:
So it won't be found and it'll be printed by number. We'll see when we get there... for now we agree it's OK?